ReactNativeTraining/src/app/screens/NavButtons.js

```diff
@@ -0,0 +1,34 @@
+import React, { PropTypes } from 'react';
+import { View, Image, Text } from 'react-native';
+
+import RoundButton from './RoundButton';
+import styles from './NavButtons.style';
+
+const NavButtons = props => {
+  const search = (
+    <Image style={styles.image} source={{ uri: 'http://www.daemconcepcion.cl/images/iconos/lupa.png' }} />
+  );
+  const add = <Text style={styles.text}>+</Text>;
+
+  return (
+    <View style={styles.container}>
+      {props.search && (
+        <RoundButton>
+          <View style={styles.search}>{search}</View>
```

Comment:
If u use `search` only here, why u declare as const outside `return` func? Put directly here. I'd declare `search` components as constant only if I repeated it several times